cmake/modules/BuildRocksDB.cmake

```diff
@@ -60,15 +60,23 @@ function(build_rocksdb)
   set(rocksdb_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/rocksdb")
   set(rocksdb_BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/rocksdb")
   set(rocksdb_LIBRARY "${rocksdb_BINARY_DIR}/librocksdb.a")
+  if(CMAKE_MAKE_PROGRAM MATCHES "make")
+    # try to inherit command line arguments passed by parent "make" job
+    set(make_cmd "$(MAKE) rocksdb")
+  else()
+    set(make_cmd "${CMAKE_COMMAND} --build <BINARY_DIR> --target rocksdb")
+  endif()
+
   ExternalProject_Add(rocksdb_ext
     SOURCE_DIR "${rocksdb_SOURCE_DIR}"
     CMAKE_ARGS ${rocksdb_CMAKE_ARGS}
     BINARY_DIR "${rocksdb_BINARY_DIR}"
-    BUILD_COMMAND ${CMAKE_COMMAND} --build <BINARY_DIR> --target rocksdb
+    BUILD_COMMAND ${make_cmd}
     BUILD_ALWAYS TRUE
     BUILD_BYPRODUCTS "${rocksdb_LIBRARY}"
     INSTALL_COMMAND "true")
 
+  unset(make_cmd)
```

Comment:
no need to `unset(make_cmd)`, as the lexical scope of the variable is limited in the function. the reason why i put `unset(make_cmd)` in `build_spdk()` was that `build_spdk()` is a macro.